content/klc/G2.1.adoc

```diff
@@ -1,21 +1,47 @@
 +++
-brief = "Defining generic and atomic parts"
+brief = "Generic, fully specified and atomic parts."
 +++
 
-In general terms, the _symbols_ in the KiCad library can be categorized into two groups:
+KiCad is flexible with how symbols and footprints can be connected. In general there are at least two workflows:
+. Generic symbol: Assign the footprint after (or during) symbol placement in eeschema.
+.. Footprint filters may be used to reduce the set of possible footprints.
+. Fully specified symbol: Assign a default footprint in the symbol.
```

Comment:
give an example, e.g. device "AP131-15", footprint "Package_TO_SOT_SMD:SOT-23-5", FPFilter: "SOT?23*", might also match "Package_TO_SOT_SMD:SOT-23-5_HandSoldering" ... maybe also mention exposed pads with and without Thermal Vias

Another example could be "LM7805_TO220" for "Package_TO_SOT_THT:TO-220-3_Vertical" with FPFIlter "TO?220*, which matches horizontal and vertical variants

Reply:
I will add more examples in the section that you would have suggested to delete. (That way it gets more informative while this part can be kept short.)